.github/workflows/stale.yml

```diff
@@ -16,16 +16,14 @@ jobs:
       - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-issue-message: "This issue is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 10 days."
-          stale-pr-message: "This PR is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 10 days."
-          close-issue-message: "This issue was closed because it has been stalled for 10 days with no activity."
-          close-pr-message: "This PR was closed because it has been stalled for 10 days with no activity."
+          stale-issue-message: "This issue is stale because it has been open 14 days with no activity. Remove stale label or comment or this will be closed in 14 days."
+          stale-pr-message: "This PR is stale because it has been open 14 days with no activity. Remove stale label or comment or this will be closed in 14 days."
+          close-issue-message: "This issue was closed because it has been stalled for 14 days with no activity."
+          close-pr-message: "This PR was closed because it has been stalled for 14 days with no activity."
           stale-issue-label: "stale"
           stale-pr-label: "stale"
-          exempt-all-assignees: true
-          exempt-all-milestones: true
+          exempt-issue-milestones: true
           exempt-issue-labels: "backlog, backlog candidate"
-          exempt-pr-labels: "backlog, backlog candidate"
           operations-per-run: 100
-          days-before-stale: 90
-          days-before-close: 10
+          days-before-stale: 14
```

Comment:
I don't think we want this to be so low. 90 days seemed good for marking as stale.

Reply:
yeah it's kinda low, but it what we put in the #1505 unless I read it wrong 🤔

Reply:
Oh, I think that was for pull requests specifically, not issues. But it looks like we can't control them separately?